opendata_module/opmon_opendata/api/views.py

```diff
@@ -392,7 +396,7 @@ def _get_harvest_rows(
     data_cursor = postgres.get_data_cursor(
         constraints=constraints, columns=columns, order_by=order_by, limit=limit, offset=offset)
 
-    return data_cursor.fetchall()
+    return data_cursor.fetchall(), columns
```

Comment:
Let's add return type hints, would be helpful

Reply:
Thanks, added